.cirrus.yml

```diff
@@ -178,7 +178,6 @@ build_and_release_dotnet_task:
     # See https://github.com/SonarSource/re-ci-images/blob/master/docker/bin/cirrus-env#L28
     - source cirrus-env QA
     - PowerShell -NonInteractive -NoProfile -File "${PIPELINE_SCRIPTS_DIR}\\build_and_pack_solution.ps1"
-    - PowerShell -NonInteractive -NoProfile -File "${PIPELINE_SCRIPTS_DIR}\\extract_project_version.ps1"
```

Comment:
the order was wrong, the variable was set here, after the begin step already run